backend/routes/event_subscriptions.ts

```diff
@@ -3,14 +3,25 @@
 import { Request, Response } from 'express';
 import { PushSubscription } from 'web-push';
```

Comment:
I think we can throw these imports into the same line
```suggestion
import webpush, { PushSubscription } from 'web-push';
```